tests/pdk/finfet/test_pinless.py

```diff
@@ -0,0 +1,26 @@
+import textwrap
+try:
+    from .utils import get_test_id, build_example, run_example
+except BaseException:
```

Comment:
This is a pretty general exception. What to we expect to go wrong here and can we restrict the exception to that failure mode.